src/main/features/claim/views/interest-continue-claiming.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('Continue to claim interest after you submit your claim?') %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {% set name = 'option' %}
+
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+
+        {{
+        radioGroup(
+          label = 'Continue to claim interest after you submit your claim?',
```

Comment:
heading and label should be in a variable really so they stay in sync

you could just use the `heading` variable here